### PR TITLE
Add dependabot automerge

### PR DESCRIPTION
**Pull request by akaszynski**
This PR adds dependabot automerge following the example described here:
https://stefanzweifel.io/posts/2021/04/28/auto-merge-dependabot-pull-requests

PRs no longer require approvals, and will merge automatically when passing.

### CODE REVIEW

File: .github/workflows/merge-me.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# ### Merge dependabot PRs ###
+# This workflow follows the example from
+# https://github.com/ridedott/merge-me-action and works by default only for PRs
+# created dependabot and will merge them when all checks pass
```
Comment (adeak) [inline .github/workflows/merge-me.yml:4]: The only aspect of this shed I can contribute to, sorry
```suggestion
# created by dependabot and will merge them when all checks pass
```

File: .github/workflows/merge-me.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    types:
+      - completed
+    workflows:
+      - 'Continuous Integration'
+
+jobs:
+  merge-me:
+    name: Merge me!
+    runs-on: ubuntu-latest
+    steps:
```
[the remark this answers is not in the document]
Comment (adeak) [reply .github/workflows/merge-me.yml:19]: If I understand correctly the access token already specifies dependabot, but it's probably a good idea to be explicit and careful.

[the remark this answers is not in the document]
Comment (tkoyama010) [reply .github/workflows/merge-me.yml:22]: I did not see the name of token `PYVISTA_CI_BOT_DEPENDABOT` 😅.

**Review by tkoyama010 — APPROVED** [review]
LGTM

**Pull request merged**